### PR TITLE
 Add lua functions for DeliverCash and DeliverExperience 

### PR DESCRIPTION
**Pull request by MustaphaTR**
Testcase adds a supply truck moving to AlliedConyard on ra shellmap.

### CODE REVIEW

File: mods/ra/maps/desert-shellmap/desert-shellmap.lua [hunk 1]
```diff
@@ -161,6 +161,17 @@ ChronoshiftAlliedUnits = function()
...
 
+SendSupplyTruck = function()
+	local truck = Actor.Create("truk", true, { Owner = allies, Facing = 64, Location = TruckEntry.Location })
+
+	truck.Move(TruckWP1.Location)
+	truck.Move(TruckWP2.Location)
+	truck.Move(TruckWP3.Location)
+	truck.DeliverCash(AlliedConYard)
+
+	Trigger.AfterDelay(DateTime.Seconds(30), SendTruck)
```
Comment (Mailaender) [inline mods/ra/maps/desert-shellmap/desert-shellmap.lua]:
```
/home/matthias/Entwicklung/OpenRA//lua/sandbox.lua:150: [string "BindingSupport.lua"]:30: Uncaught CLR exception at Lua->CLR boundary: System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.Reflection.TargetInvocationException: Exception has been thrown by the target of an invocation. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at OpenRA.Mods.Common.Scripting.TriggerGlobal.AfterDelay (System.Int32 delay, Eluant.LuaFunction func) [0x00015] in <9625bc0a717d47ee85400cd20db8a379>:0
```

Comment (Mailaender) [reply mods/ra/maps/desert-shellmap/desert-shellmap.lua]: Works fine when fixing the oversight: 

```patch
-Trigger.AfterDelay(DateTime.Seconds(30), SendTruck)
+Trigger.AfterDelay(DateTime.Seconds(30), SendSupplyTruck)
```

**Review by Mailaender — DISMISSED** [review]

**Review by reaperrr — DISMISSED** [review]

**Comment by reaperrr** [comment]
Looks good to me, testcase can be removed now.

**Comment by MustaphaTR** [comment]
Removed testcase.

**Review by reaperrr — APPROVED** [review]

**Comment by reaperrr** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/1d71f388b6128557f71243c7a452b8bdb4407ff5...0416f4b379717409506c6ab1b5ef53d4cdc5b2a2

**Pull request merged**